.gitmodules

```diff
@@ -6,3 +6,6 @@
 	path = cert-manager-operator
 	url = https://github.com/openshift/cert-manager-operator.git
 	branch = master
+[submodule "istio-csr"]
+	path = istio-csr
+	url = https://github.com/openshift/cert-manager-istio-csr
```

Comment:
Please set branch for the sub-module using `git submodule set-branch --branch main istio-csr` command.
Sorry missed this input.

Reply:
Added Thanks!